internal/server/scanner.go

```diff
@@ -337,13 +335,11 @@ func (sw *scanWriter) Step(n uint64) {
 
 // ok is whether the object passes the test and should be written
 // keepGoing is whether there could be more objects to test
-func (sw *scanWriter) testObject(id string, o geojson.Object, fields []float64, ignoreGlobMatch bool) (
+func (sw *scanWriter) testObject(id string, o geojson.Object, fields []float64) (
 	ok, keepGoing bool, fieldVals []float64) {
-	if !ignoreGlobMatch {
-		match, kg := sw.globMatch(id, o)
-		if !match {
-			return false, kg, fieldVals
-		}
+	match, kg := sw.globMatch(id, o)
```

Comment:
Why drop `ignoreGlobMatch`? Seems like we were avoiding `sw.globMatch` in most cases.

Reply:
This was only needed because of the dual filtering in nearestNeighbors, so not needed now.